samples/maps/demo/custom-map/demo.details

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
This concept is already demonstrated in `maps/series/data-map` so this sample can be deleted and the docs updated.

Reply:
I'm sorry, but I couldn't find `maps/series/data-map` or any demo similar to this one. What should I do here?

Reply:
Sorry, I meant `maps/series/data-path`